sklearn/utils/estimator_checks.py

```diff
@@ -293,7 +293,9 @@ def set_testing_parameters(estimator):
     if "decision_function_shape" in params:
         # SVC
         estimator.set_params(decision_function_shape='ovo')
-
+    if "n_best" in params:
```

Comment:
``n_best`` should be set to ``n_components`` by default, I think. Seems like a usability issue in the class.